connectors/sources/generic_database.py

```diff
@@ -36,6 +38,45 @@ def table_filter(table):
     )
 
 
+class Queries(ABC):
```

Comment:
Nice! Looks way better IMO. Could we move this change and the corresponding changes in the specific data sources to a separate PR? Would make it easier to review and it's more explicit in the git history.

Reply:
Hey @timgrein, I have separated the logic of Queries class in PR #565